lib/reverse_sentence.rb

```diff
@@ -1,6 +1,34 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n), where n is the length of a sentence.
+# Space complexity: O(n)
```

Comment:
Can we improve on this by doing our operations _in-place_? (i.e. modifying `my_sentence` instead of creating a separate `new_sentence` variable)